src/pages/basic_data/app_admin/UserList.tsx

```diff
@@ -100,68 +62,29 @@ export const UserList = ({ userList, tableCaption, roleToRemove, roleToAdd, refe
-                    <TableCell width="150">{t('common.actions')}</TableCell>
-                  </TableRow>
-                </TableHead>
-                <TableBody>
-                  {adminsOnPage.map((user) => {
-                    const isLoading = updatedRoleForUsers.includes(user.username);
-                    const disableAddButton = user.roles.some((role) => role.rolename === roleToAdd);
-                    return (
-                      <TableRow key={user.username}>
-                        <TableCell>{user.username}</TableCell>
-                        <TableCell>
-                          {user.givenName} {user.familyName}
-                        </TableCell>
-                        <TableCell>
-                          {roleToRemove && (
-                            <Button
-                              color="error"
-                              variant="outlined"
-                              startIcon={<CancelIcon />}
-                              disabled={isLastInstitutionAdmin}
-                              data-testid={`button-remove-role-${roleToRemove}-${user.username}`}
-                              onClick={() => setRemoveRoleForUser(user.username)}>
-                              {t('common.remove')}
-                            </Button>
-                          )}
-                          {roleToAdd && (
-                            <LoadingButton
-                              variant="contained"
-                              size="small"
-                              startIcon={<AddIcon />}
-                              loadingPosition="start"
-                              disabled={disableAddButton}
-                              loading={!disableAddButton && isLoading}
-                              data-testid={`button-add-role-${roleToAdd}-${user.username}`}
-                              onClick={() => handleAddRoleToUser(user)}>
-                              {t('common.add')}
-                            </LoadingButton>
-                          )}
-                        </TableCell>
-                      </TableRow>
-                    );
-                  })}
-                </TableBody>
-              </Table>
-            </TableContainer>
-          </ListPagination>
+          <List>
+            {sortedList.map((user) => (
+              <ListItem
+                sx={{ backgroundColor: 'white' }}
+                key={user.username}
+                divider
+                secondaryAction={
+                  <IconButton
+                    aria-label={t('common.remove')}
+                    color="primary"
+                    disabled={isLastInstitutionAdmin}
+                    data-testid={`button-remove-role-${roleToRemove}-${user.username}`}
+                    onClick={() => setRemoveRoleForUser(user.username)}>
+                    <CancelIcon />
+                  </IconButton>
+                }>
+                <ListItemAvatar>
```

Comment:
Ikke verdens viktigste sak for en såpass intern sak, men fint å være bevisst på a11y på alt. mangler noe aria-label her. Det beste er kanskje å ha en `id` på hele navnet, og en `aria-labelledby`?
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/ccf331ce-e5f3-4a7e-a19f-cbce5f705faa)

Reply:
Dette gjelder `alt` text ser jeg nå ja, så kanskje bare sette `alt=''` da, om det er bra nok 🤔

Reply:
Kan ikke mate inn alt til div i react. Er kun rolle img på disse.

Aria-labellby er forøvrig ikke bra nok med wcag mener jeg? kravet er vel at alle bilder skal ha en alt-tag.
Forsøkte med aria-hidden og det var ikke bra nok. 

Alt tags er ikke kun for svaksynte men også når bilder ikke laster inn korrekt.

Reply:
trodde det var å bare legge til `alt` på img taggen her, men det var en liten jungel ser jeg. Dette er bare for 2-3personer ansatt i Sikt, så tenker vi kan se gjennom fingrene for denne gangen :))
Vi har en generell oppgave i loopen for å fikse på personvisning på alle sider, ser at Joachim også styrer med lignende vising av Person, og det vi har fra før på dette er ikke helt 100% dessverre